event.c

```diff
@@ -350,8 +350,13 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
             void *dummy2 ATTRIBUTE_UNUSED, struct tm *currenttime_tm)
 {
     char fullfilename[PATH_MAX];
-
-    if (strcmp(cnt->conf.snappath, "lastsnap")) {
+    int len = strlen(cnt->conf.snappath);
+    int offset = 0;
+    if (len >= 9)
```

Comment:
sort of magic, the length of  lastsanp is 8 and I only need to play with it
if it has stuff on the front.

*Jim Dodgen*







On Sat, May 16, 2015 at 1:13 PM, tosiara <notifications@github.com> wrote:

> In event.c
> <https://github.com/Mr-Dave/motion/pull/73#discussion_r30464193>:
>
> > @@ -350,8 +350,13 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
> >              void *dummy2 ATTRIBUTE_UNUSED, struct tm *currenttime_tm)
> >  {
> >      char fullfilename[PATH_MAX];
> > -
> > -    if (strcmp(cnt->conf.snappath, "lastsnap")) {
> > +    int len = strlen(cnt->conf.snappath);
> > +    int offset = 0;
> > +    if (len >= 9)
>
> Can you explain why do you check for "9"?
> Is that some magic number?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mr-Dave/motion/pull/73/files#r30464193>.
>